doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst

```diff
@@ -104,6 +104,412 @@ Thingy:91 has a multimode modem, which enables it to support automatic switching
+
+      #. Navigate to where you extracted the firmware.
+
+      #. Open the folder :file:`img_fota_dfu_hex` that contains the HEX files for updating over USB.
+         See the :file:`CONTENTS.txt` file for information on which file you need.
+
+      #. Select the appropriate Asset Tracker v2 firmware file.
+
+         .. note::
+
+            If you are connecting over NB-IoT and your operator does not support extended Protocol Configuration Options (ePCO), select the file that has legacy Protocol Configuration Options (PCO) mode enabled.
+
+      #. Click :guilabel:`Open`.
+
+      #. Scroll down in the menu on the left to the **DEVICE** section and click :guilabel:`Write`.
+
+         .. figure:: images/programmer_hex_write.png
+            :alt: Programmer - Writing of HEX files
+
+            Programmer - Writing of HEX files
+
+         The MCUboot DFU window appears.
+
+         .. figure:: images/thingy91_mcuboot_dfu1.png
+            :alt: Programmer - MCUboot DFU
+
+            Programmer - MCUboot DFU
+
+      #. In the **MCUboot DFU** window, click :guilabel:`Write`.
+         When the update is complete, a **Completed successfully** message appears.
+      #. Scroll up in the menu on the left to the **FILE** section and click :guilabel:`Clear files`.
+
+   .. group-tab:: Through external debug probe
+
+      To program the application firmware using external debug probe, complete the following steps:
+
+      1. Open `nRF Connect for Desktop`_ and launch the Programmer app and :ref:`prepare the hardware <prepare_hw_ext_dp>` if you have not done it already.
+      #. Make sure the SWD selection switch **SW2** is set to **nRF91** on the Thingy:91.
+
+      #. In the menu on the left, click :guilabel:`Add file` in the **FILE** section, and select :guilabel:`Browse`.
+         A file explorer window appears.
+
+         .. figure:: images/programmer_add_file1.png
+            :alt: Programmer - Add file
+
+            Programmer - Add file
+
+      #. Navigate to where you extracted the firmware.
+
+      #. Open the folder :file:`img_app_bl` that contains the HEX files for updating using a debugger.
+         See the :file:`CONTENTS.txt` file for information on which file you need.
+
+      #. Select the appropriate Asset Tracker v2 firmware file.
+
+         .. note::
+
+            If you are connecting over NB-IoT and your operator does not support extended Protocol Configuration Options (ePCO), select the file that has legacy Protocol Configuration Options (PCO) mode enabled.
+
+      #. Click :guilabel:`Open`.
+      #. Scroll down in the menu on the left to the **DEVICE** section and click :guilabel:`Erase & write`.
+         The update is completed when the animation in Programmer's **Device memory layout** window ends.
+
+         .. figure:: images/programmer_ext_debug_hex_write.png
+            :alt: Programming using External debug probe
+
+            Programming using External debug probe
```

Comment:
There are a couple instances of this capitalization. Not sure if there is a reason. I'd also prefer 'an external debug probe'

Reply:
Fixed.